cf/cmd/cli/commands/provider/provider.go

```diff
@@ -19,11 +30,72 @@ var BootstrapCommand = cli.Command{
 	Name:        "bootstrap",
 	Description: "bootstrap a provider from the registry",
 	Usage:       "bootstrap a provider from the registry",
+	Flags: []cli.Flag{
+		&cli.StringFlag{Name: "bootstrapBucket", Aliases: []string{"bb"}, Usage: "The name of the bootstrap bucket provider is deployed to", EnvVars: []string{"DEPLOYMENT_BUCKET"}},
+	},
 	Action: func(c *cli.Context) error {
 		id := c.Args().First()
 		if id == "" {
 			return errors.New("id argument must be provided")
 		}
+
+		var cfg config.ProviderDeploymentCLI
```

Comment:
Rather than loading config from a .env file to get the registry API URL

We will want to use internal/build/urls.go

```
package build

// these URLs are overridden as part of the release build process.
var (
	ProviderRegistryAPIURL = "localhost:9091"
)
```

at build time when we release this CLI, we will us `-ldflags` to overwrite the value of this variable with the production registry URL

To see this in action, we use this technique in the Granted CLI repo